src/py/mat3ra/made/tools/build/perturbation/builders.py

```diff
@@ -0,0 +1,91 @@
+from typing import List, Optional, Any
+
+from mat3ra.made.material import Material
+from mat3ra.made.tools.build import BaseBuilder
+
+from .configuration import PerturbationConfiguration
+from ...modify import wrap_material, translate_to_z_level
+from ...utils import PerturbationFunctionHolder
+
+
+class PerturbationBuilder(BaseBuilder):
+    _ConfigurationType: type(PerturbationConfiguration) = PerturbationConfiguration  # type: ignore
+    _GeneratedItemType: Material = Material
+    _PostProcessParametersType: Any = None
+
+    @staticmethod
+    def _prepare_material(configuration):
```

Comment:
Don't we have _pre_process or something like that?

Reply:
No, we didn't implement that